kubernetes-client/src/main/java/io/fabric8/kubernetes/client/handlers/KubernetesListHandler.java

```diff
@@ -57,7 +57,7 @@ public String getApiVersion() {
 
   @Override
   public KubernetesList create(OkHttpClient client, Config config, String namespace, KubernetesList item, boolean dryRun) {
-    return new KubernetesListOperationsImpl(client, config, namespace, null, DeletionPropagation.BACKGROUND, false, false, item, null, dryRun).create();
+    return new KubernetesListOperationsImpl(client, config, namespace, null, DeletionPropagation.BACKGROUND, -1, false, false, item, null, dryRun).create();
```

Comment:
If there is a constant for default values (propagation policy, grace period), we should use those